### PR TITLE
Update and add to documentation

### PR DESCRIPTION
**Pull request by ldv8434**
Changes proposed in this pull request:

- fix outdated feed-adding documentation
- fix dead links
- remove mailing list info as it does not exist anymore
- add index page to developer section so that it is easier to get to the other pages
- add user documentation for main view
- add dedicated page for bug reports
- move subscription management information from global config page to subscription management page
- add documentation for subscription management 

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [x] documentation updated

Please let me know if this is up to standard. This is my first contribution to a project and I'm not familiar with a lot of the rules and such, but I've done my best with the documentation that existed.

### CODE REVIEW

File: docs/en/developers/06_Reporting_Bugs.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ 3. If your request is new, [open a new bug ticket](https://github.com/FreshRSS/FreshRSS/issues/new)
+ 4. Finally, write your request. If you're fluent in English, it's the preferred language because it allows for discussion with the largest number of people.
+ 5. Please follow the tips below to make it easier to let your ticket be heard.
+ 
+## Informal
+
+Not everyone likes or uses GitHub for a variety of legitimate reasons. That is why you can also contact us in a more informal way.
+
+* On [our Mattermost chat](https://framateam.org/signup_user_complete/?id=e2680d3e3128b9fac8fdb3003b0024ee)
+* On [the mailing lists](https://freshrss.org/announce-of-the-mailing-lists.html)
```
Comment (aledeg) [inline docs/en/developers/06_Reporting_Bugs.md]: Maybe this should be removed as well!

File: docs/en/users/03_Main_view.md [hunk 2]
```diff
@@ -1,14 +1,36 @@
+FreshRSS support has three primary viewing modes: Normal, Global, and Reader view.
```
Comment (aledeg) [inline docs/en/users/03_Main_view.md]: `FreshRSS supports` or `FreshRSS has` but not both :)

File: docs/en/users/03_Main_view.md [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+* **Statistics:** View statistics about the feed
+* **See website:** Open the feed's website in another tab
+* **Manage:** Configure the feed
+* **Actualize:** Force-update the feed
+* **Mark as read:** Mark all items in the feed as read
 
 # Global view
 
-**TODO**
+Global view allows quick views of many feeds' statuses at once. Feeds and categories are shown with the number of unread articles next to them. Clicking a stream will open it in a view similar to normal view.
```
Comment (aledeg) [inline docs/en/users/03_Main_view.md]: `feed's statuses` instead of `many feeds' statuses` and replace `Clicking a stream` by `Clicking on a feed's name`!

File: docs/en/users/03_Main_view.md [hunk 4]
```diff
@@ -1,14 +1,36 @@
...
 
 # Global view
 
-**TODO**
+Global view allows quick views of many feeds' statuses at once. Feeds and categories are shown with the number of unread articles next to them. Clicking a stream will open it in a view similar to normal view. 
 
 # Reader view
 
-**TODO**
+Reader view will display a stream will all articles already open for reading. Streams can be switched by clicking the folder icon at the top to bring up the category/feed sidebar.
```
Comment (aledeg) [inline docs/en/users/03_Main_view.md]: Here, replace `stream` by `feed`.

File: docs/en/users/04_Subscriptions.md [hunk 5]
```diff
@@ -1,11 +1,90 @@
 # Adding a feed
 
  1. To add a feed, copy the URL of its RSS or Atom file (for instance, the Framablog RSS URL is `https://framablog.org/feed/`). FreshRSS is able to automatically find the address of the feed for websites that are declaring it in a standard way.
- 2. On FreshRSS, click “Subscriptions management”.
- 3. Paste the URL in “Add an RSS feed” below the page title.
+ 2. On FreshRSS, click the plus button next to “Subscriptions management”.
```
Comment (aledeg) [inline docs/en/users/04_Subscriptions.md]: Maybe you could add the plus icon in this sentence.

File: docs/en/users/04_Subscriptions.md [hunk 6]
```diff
@@ -1,11 +1,90 @@
...
+ 3. Paste the URL in “Feed URL” below the page title.
```
Comment (aledeg) [inline docs/en/users/04_Subscriptions.md]: You can remove `below the page title` since this has changed. I would rephrase it `Paste the URL in the “Feed URL” field.`

File: docs/en/users/04_Subscriptions.md [hunk 7]
```diff
@@ -1,11 +1,90 @@
...
  4. (optional): You can select the category for your feed. By default, it will be in “Uncategorized”.
```
Comment (aledeg) [inline docs/en/users/04_Subscriptions.md:6]: You could add also the other optional option too

File: docs/en/users/04_Subscriptions.md [hunk 8]
```diff
@@ -1,11 +1,90 @@
...
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<h2>wanted</h2>
+<p class='ad'>unwanted content</p>
+<p class='content'>wanted content</p>
+</div>
+```
+In this case it's possible to combine multiple CSS selectors with a comma: ```#article p.content, #article h2```
+
+#### Retrieve a truncated stream with external tools
```
Comment (aledeg) [inline docs/en/users/04_Subscriptions.md]: `feed` instead of `stream`

File: docs/en/users/04_Subscriptions.md [hunk 9]
```diff
@@ -1,11 +1,90 @@
...
+
+Complimentary tools can be used to retrieve full article content, such as:
+
+* [RSS-Bridge](https://github.com/RSS-Bridge/rss-bridge)
+* [Full-Text RSS](https://bitbucket.org/fivefilters/full-text-rss)
```
Comment (aledeg) [inline docs/en/users/04_Subscriptions.md:83]: That's nice to mention other tools :+1:

**Review by aledeg — CHANGES_REQUESTED** [review]

**Comment by aledeg** [comment]
@ldv8434 Thank you for your contribution! That's greatly appreciated.
Could you address my comments regarding the minor changes I've requested?
Good job :+1:

File: docs/en/developers/06_Reporting_Bugs.md (hunk 1 above)
Comment (ldv8434) [reply docs/en/developers/06_Reporting_Bugs.md]: The Mattermost chat seemed to still be in use, so I only removed the mailing list line. Please let me know if this is not accurate.

Comment (aledeg) [reply docs/en/developers/06_Reporting_Bugs.md]: To be honest, I've never used the Mattermost chat. So I wouldn't know :)

File: docs/en/users/04_Subscriptions.md [hunk 10]
```diff
@@ -1,11 +1,93 @@
...
 
  1. To add a feed, copy the URL of its RSS or Atom file (for instance, the Framablog RSS URL is `https://framablog.org/feed/`). FreshRSS is able to automatically find the address of the feed for websites that are declaring it in a standard way.
- 2. On FreshRSS, click “Subscriptions management”.
- 3. Paste the URL in “Add an RSS feed” below the page title.
+ 2. On FreshRSS, click the "**+**" button next to “Subscriptions management”.
+ 3. Paste the URL in “Feed URL” field.
  4. (optional): You can select the category for your feed. By default, it will be in “Uncategorized”.
+ 5. (optional): If the subscription requires credentials, you can enter them in the "HTTP username" and "HTTP password" fields. 
+ 6. (optional): You can get a timeout for the feed request if the feed requires it.
+ 7. (optional): You can choose to ignore SSL certificate errors (such as with self-signed certificates) by setting "Verify SSL security" to "No". This is not recommended, and it is better to either add the root certificate to the FreshRSS server or to fix the SSL certificate problems on the feed hosting server.
```
Comment (aledeg) [inline docs/en/users/04_Subscriptions.md]: :+1:

**Review by aledeg — APPROVED** [review]
LGTM

File: docs/en/contributing.md [hunk 11]
```diff
@@ -25,6 +18,8 @@ We also need some information:
 * Your storage system (SQLite, MySQL, MariaDB, PostgreSQL)
 * If possible, the related logs (PHP logs and FreshRSS logs under `data/users/your_user/log.txt`)
 
+For a more detailed guide on reporting bugs, please refer to [the appropriate page](developers/06_Reporting_Bugs).
```
Comment (Frenzie) [inline docs/en/contributing.md]: I don't care for the word appropriate here.
```suggestion
For a more detailed guide on writing bug reports, please refer to [the in-depth guide on reporting bugs](developers/06_Reporting_Bugs).
```

File: docs/en/developers/06_Reporting_Bugs.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
+# Reporting a bug or a suggestion
+
+Despite the care given to FreshRSS, it's still possible that bugs occur. The project is young and development is dynamic, so it can be corrected quickly. You might also have a feature in mind that doesn't yet exist. Regardless whether your idea seems silly, far-fetched, useless or too specific, please don't hesitate to propose it to us! "Ideas in the air" often find an attentive ear. It's new external perspectives that make the project evolve the most.
```
Comment (Frenzie) [inline docs/en/developers/06_Reporting_Bugs.md]: Is the project still young? :-P
```suggestion
Despite the care given to FreshRSS, it's still possible that bugs occur. Development is dynamic, so issues can be corrected quickly. You might also have a feature in mind that doesn't yet exist. Regardless whether your idea seems silly, far-fetched, useless or too specific, please don't hesitate to propose it to us! "Ideas in the air" often find an attentive ear. It's new external perspectives that make the project evolve the most.
```

File: docs/en/users/04_Subscriptions.md [hunk 13]
```diff
@@ -1,11 +1,93 @@
 # Adding a feed
 
  1. To add a feed, copy the URL of its RSS or Atom file (for instance, the Framablog RSS URL is `https://framablog.org/feed/`). FreshRSS is able to automatically find the address of the feed for websites that are declaring it in a standard way.
- 2. On FreshRSS, click “Subscriptions management”.
- 3. Paste the URL in “Add an RSS feed” below the page title.
+ 2. On FreshRSS, click the "**+**" button next to “Subscriptions management”.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 2. In FreshRSS, click the "**+**" button next to “Subscriptions management”.
```

File: docs/en/users/04_Subscriptions.md [hunk 14]
```diff
@@ -1,11 +1,93 @@
...
+ 3. Paste the URL in “Feed URL” field.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 3. Paste the URL in the “Feed URL” field.
```

File: docs/en/users/04_Subscriptions.md [hunk 15]
```diff
@@ -1,11 +1,93 @@
...
  4. (optional): You can select the category for your feed. By default, it will be in “Uncategorized”.
+ 5. (optional): If the subscription requires credentials, you can enter them in the "HTTP username" and "HTTP password" fields. 
+ 6. (optional): You can get a timeout for the feed request if the feed requires it.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 6. (optional): You can set a timeout for the feed request if the feed requires it.
```

File: docs/en/users/04_Subscriptions.md (hunk 10 above)
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
 7. (optional): You can choose to ignore SSL certificate errors (such as with self-signed certificates) by setting "Verify SSL security" to "No". This is not recommended, and it is better to either add the root certificate to the FreshRSS server or to fix the SSL certificate problems on the feed hosting server.
```

File: docs/en/users/04_Subscriptions.md [hunk 16]
```diff
@@ -1,11 +1,93 @@
...
+# Subscription management
+
+The "Subscription management" submenu allows categories and feeds to be configured. Feeds can be moved between categories by drag-and-drop, or in the individual feed's settings. Hovering over a feed/category will cause a gear icon to appear. Clicking the icon will bring up the settings for that item. 
+
+## Category Settings
+
+### Information 
+
+* **Title:** Name of category
+* **Display position:** Defines order of categories. Lower numbers get priority, non-numbered items come last, and equally numbered items will sort by alphabetical order.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]: There has to be a better way of saying that numbers are counted naturally, but I haven't fully woken up yet. ^_^
```suggestion
  * **Display position:** Defines the order of categories. Lower numbers get priority, non-numbered items come last, and equally numbered items will sort by alphabetical order.
```

File: docs/en/users/04_Subscriptions.md [hunk 17]
```diff
@@ -1,11 +1,93 @@
...
+
+### Archiving
+
+If "Purge Policy" has "By default" selected, then the [default purge policy](./05_Configuration.md) is used and the other options are not displayed. Category options will override the default policy, but will not override feed-specific options.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
If "Purge Policy" has "By default" selected, then the [default purge policy](./05_Configuration.md) is used and the other options are not displayed. Category options will override the default policy, but they will not override feed-specific options.
```

File: docs/en/users/04_Subscriptions.md [hunk 18]
```diff
@@ -1,11 +1,93 @@
...
+
+## Feed Settings
+
+### Information
+
+These should be auto-filled when adding a feed, but can be modified after. **Visibility** will define if the feed is displayed in the main feed, only in category feeds, or not at all.
+
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]: It's generally preferred not to use subheadings immediately after another (sub)heading), and to open with a text very much like this instead! ;-)
```suggestion

These fields will be auto-filled when adding a feed, but they can be modified later. **Visibility** will define if the feed is displayed in the main feed, only in specific categories, or not at all.

```

File: docs/en/users/04_Subscriptions.md [hunk 19]
```diff
@@ -1,11 +1,93 @@
...
+### Advanced
+
+#### Retrieve a truncated feed from within FreshRSS
+
+This question comes up regularly, so we'll try to clarify how one can retrieve a truncated RSS feed with FreshRSS. Please note that the process is absolutely not user friendly, but it works. :)
+
+Please be aware that this way you'll generate much more traffic to the originating sites, and they might block you accordingly. FreshRSS performance is also negatively affected, because you'll have to fetch the full article content one by one. So it's a feature to use sparingly!
+
+What's meant by "CSS path of articles on the original site" actually corresponds to the "path" consisting of IDs and classes (which in HTML, matches the id and class attributes) to retrieve only the interesting part that corresponds to the article. Ideally, this path starts with an id (which is unique to the page).
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]: This is outdated. It now says `Article CSS selector on original website` because, well, that's what it is and you can actually use a search engine with those terms.

File: docs/en/users/04_Subscriptions.md [hunk 20]
```diff
@@ -1,11 +1,93 @@
...
+<h2>wanted</h2>
+<p class='ad'>unwanted content</p>
+<p class='content'>wanted content</p>
+</div>
+```
+In this case it's possible to combine multiple CSS selectors with a comma: ```#article p.content, #article h2```
+
+#### Retrieve a truncated feed with external tools
+
+Complimentary tools can be used to retrieve full article content, such as:
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]: Complimentary means that RSS-Bridge keeps saying what a nice application FreshRSS is. :-P
```suggestion
Complementary tools can be used to retrieve full article content, such as:
```

File: docs/en/users/04_Subscriptions.md [hunk 21]
```diff
@@ -1,11 +1,93 @@
...
+
+##### Add the corresponding classes to the article CSS path on the feed configuration page. Examples:
+
+*  Rue89: ```#article .content```
+*  PCINpact: ```#actu_content```
+*  Lesnumériques: ```article#body div.text.clearfix```
+*  Phoronix: ```#main .content```
+
+##### Combining CSS Classes
+Let's say we have an article which contains ads within its content, and we do not want to have those ads retrieved by FreshRSS. Example Source Code:
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
Let's say we have an article which contains ads, and we do not want to have those ads retrieved by FreshRSS. Example HTML:
```

File: docs/en/users/04_Subscriptions.md [hunk 22]
```diff
@@ -1,11 +1,93 @@
...
+<div id='article'>
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<p class='ad'>unwanted content</p>
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<h2>wanted</h2>
+<p class='ad'>unwanted content</p>
+<p class='content'>wanted content</p>
+</div>
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]: Better to use double quotes to be consistent with elsewhere in the docs.

File: docs/en/users/04_Subscriptions.md [hunk 23]
```diff
@@ -1,11 +1,93 @@
...
+#### Retrieve a truncated feed with external tools
+
+Complimentary tools can be used to retrieve full article content, such as:
+
+* [RSS-Bridge](https://github.com/RSS-Bridge/rss-bridge)
+* [Full-Text RSS](https://bitbucket.org/fivefilters/full-text-rss)
+
+### Filter
+
+Articles can be automatically marked as read based on some search terms. See [filtering](./03_Main_view.md#filtering-articles) for how to create these filters.
```
Comment (Frenzie) [inline docs/en/users/04_Subscriptions.md]:
```suggestion
Articles can be automatically marked as read based on some search terms. See [filtering](./03_Main_view.md#filtering-articles) for more information on how to create these filters.
```

File: docs/en/users/05_Configuration.md [hunk 24]
```diff
@@ -66,11 +66,27 @@ If you don't use those buttons because you never browse on mobile or because you
 
 # Reading
 
-**TODO**
+## Number of articles per page
+
+Defines the number of articles to display at once before needing to load more. In normal and reading view, more articles are loaded automatically. In global view, a button will appear at the bottom of the loaded list.
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]:
```suggestion
This setting defines the number of articles to display at once before needing to load more. In normal and reading view, more articles are loaded automatically. In global view, a button will appear at the bottom of the list. 
```

**Review by Frenzie — COMMENTED** [review]
Cheers :+1:

File: docs/en/developers/06_Reporting_Bugs.md (hunk 1 above)
Comment (Frenzie) [reply docs/en/developers/06_Reporting_Bugs.md]: We miss you on there! ;-D

(To be honest I don't drop by much but I do think it can be useful.)

File: docs/en/developers/06_Reporting_Bugs.md [hunk 25]
```diff
@@ -0,0 +1,65 @@
...
+ 3. If your request is new, [open a new bug ticket](https://github.com/FreshRSS/FreshRSS/issues/new)
+ 4. Finally, write your request. If you're fluent in English, it's the preferred language because it allows for discussion with the largest number of people.
+ 5. Please follow the tips below to make it easier to let your ticket be heard.
+ 
+## Informal
+
+Not everyone likes or uses GitHub for a variety of legitimate reasons. That is why you can also contact us in a more informal way.
+
+* On [our Mattermost chat](https://framateam.org/signup_user_complete/?id=e2680d3e3128b9fac8fdb3003b0024ee)
+* At events / meetings around Free Software
```
Comment (aledeg) [inline docs/en/developers/06_Reporting_Bugs.md:23]: There is also a subreddit https://www.reddit.com/r/freshrss/. Not very active but it works also.

Comment (Alkarex) [reply docs/en/developers/06_Reporting_Bugs.md:23]: Ah, I did not know / had forgotten :-P

File: docs/en/admins/03_Installation.md [hunk 26]
```diff
@@ -10,14 +10,16 @@ Before you begin, make sure that you've read the [prerequisites](02_Prerequisite
 
 3. Give ownership of the FreshRSS folder to your web server user (often `www-data`). Give group read permissions to all files in `.`[^2], and group write permissions to `./data/`.
 
-4. Install needed PHP modules.
+4. Install needed PHP modules: libxml, cURL, JSON, PDO_MySQL, PCRE, and ctype. (Zlib, mbstring, iconv, and ZipArchive are recommended as well).
```
Comment (Alkarex) [inline docs/en/admins/03_Installation.md]: We can refer to our Dockerfile for Debian and Alpine for a precise and up-to-date list:
* https://github.com/FreshRSS/FreshRSS/blob/master/Docker/Dockerfile#L11-L12
* https://github.com/FreshRSS/FreshRSS/blob/master/Docker/Dockerfile-Alpine#L7-L9

File: docs/en/users/04_Subscriptions.md (hunk 19 above)
Comment (ldv8434) [reply docs/en/users/04_Subscriptions.md]: I hadn't written this section - just moved it to a new place that made more sense. What would you recommend here? I can change the terms or try to re-write the section.

Comment (Frenzie) [reply docs/en/users/04_Subscriptions.md]: I'm not quite sure. Let's just leave it more or less the same for now?
```suggestion
The  "Article CSS selector on original website" corresponds to the "path" consisting of IDs and classes (which in HTML, matches the id and class attributes) to retrieve only the interesting part that corresponds to the article. Ideally, this path starts with an id (which is unique to the page). The basics are explained [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors).
```

File: docs/en/index.md [hunk 27]
```diff
@@ -18,7 +18,7 @@ FreshRSS has a lot of features including:
 
 This documentation is split into different sections:
 
-* [User documentation](./users/02_First_steps.html), where you can discover all the possibilities offered by FreshRSS
-* [Administrator documentation](./admins/01_Index.html) for detailed installation and maintenance related tasks
-* [Developer documentation](./developers/01_First_steps.html) to guide you in the source code of FreshRSS and to help you if you want to contribute
+* [User documentation](./users/02_First_steps.md), where you can discover all the possibilities offered by FreshRSS
```
Comment (Frenzie) [inline docs/en/index.md:21]: You can't do that. ;-)

https://freshrss.github.io/FreshRSS/en/users/02_First_steps.html
https://freshrss.github.io/FreshRSS/en/users/02_First_steps.md

Comment (ldv8434) [reply docs/en/index.md:21]: Github Pages' uses the jekyll-relative-links plugin to convert relative ".md" links to ".html". Should be good to go! 

See:
https://github.com/FreshRSS/FreshRSS/blob/master/docs/en/admins/01_Index.md
https://freshrss.github.io/FreshRSS/en/admins/01_Index.html

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
Thanks @ldv8434 👍🏻 
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md
